contracts/Comptroller.sol

```diff
@@ -165,7 +165,7 @@ contract Comptroller is
 
         uint256 accountAssetsLen = accountAssets[msg.sender].length;
 
-        _ensureMaxLoops(accountAssetsLen);
+        _ensureMaxLoops(accountAssetsLen + len);
```

Comment:
By doing this, I suppose we can remove the previous check `_ensureMaxLoops(len);`

Reply:
Fixed in https://github.com/VenusProtocol/isolated-pools/pull/202